app/views/layouts/application.html.erb

```diff
@@ -8,6 +8,9 @@
     <%= csrf_meta_tags %>
     <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet"/>
+    <!-- The following line is a side fix to work around our issue with @import url(..) declarations in Sass. -->
+    <!-- As soon as possible, the Sass issues should be fixed and this streamlined back into our Sass build. -->
```

Comment:
Can we add a Jira ticket here?

Reply:
Created on Jira.

Reply:
The idea was to add the ticket to the comments :)